### ISSUE TITLE
Citybase Implementation Worksheet Updates

### ISSUE THREAD

**Issue by moorerst** [cityofaustin/atd-data-tech#9915]
Standard Checkout Worksheet updates following Discovery meeting:

•	Document the expected description attribute shown to the customer on the Front End (we have listed the WIP version that Diana posted in the chat yesterday as a point of reference)
•	Document the expected sub-description attribute shown to the customer on the Front End (we have listed the WIP version that Diana posted in the chat yesterday as a point of reference)
•	List and finalize the back end attributes required to drive internal processes

The output would look something like this (This is a representative example from APH worksheet):

 ![image.png](https://images.zenhubusercontent.com/5da61135c9a7f8000179986e/b6408fd4-e4d0-4cb1-a0f9-d1ead9894dbd)

**Comment by moorerst** [comment]
meeting with stakeholders 8/3 to review suggestions for invoice attributes

![image](https://user-images.githubusercontent.com/56606465/182682392-d2b4c40e-2e5e-4903-94ad-8a15f71f079f.png)

![image](https://user-images.githubusercontent.com/56606465/182682473-fdc65967-245d-42ac-abb9-c6dd087c8478.png)

work sessions scheduled 8/8 with Diana to update the worksheet and resubmit to Citybase

**Comment by moorerst** [comment]
met with stakeholders and they would like to have the following information added to the invoice:
Install Date
Locations: Districts

Diana mocked up several ideas based on what the stakeholders requested:

**Comment by moorerst** [comment]
@dianamartin in the updated description/sub description table should **District** and **Unit Cost** be added as Sub Descriptions to appear on the invoice?  
https://docs.google.com/document/d/1TBzIQOXBY_3bNbiBIqw9pfJPzIe7mrmKY4e4ldMLAcY/edit

**Comment by dianamartin** [comment]
@moorerst I realized we didn't add the banner names, I had just mocked it up on that prototype receipt. I'm trying to get it going in the CityBase test environment now.

**Comment by dianamartin** [comment]
Taejin emailed on 8/10
>Hi all, 
Checking in here to see how the ATD team was progressing on locking down the custom attributes in light of our discussion last week.

>Let us know if there is anything we can do to help.

I had worked on some changes to the transactions table, I synced with John and Stephanie on 8/11 to see what alternatives we had. We think we've got a solution to the Fee Description and Sub Description. 

Looks like this:
![image](https://user-images.githubusercontent.com/21112975/184249390-786a9ad5-825f-4244-8874-4a957252feb1.png)

Emailed Citybase back 8/11
>We have been working through some data structure things on our end but think we’re at a good point now.

>Here is the updated table with the custom attributes. We mocked up what that Fee Description and Sub Description might look like in the system. 
 
![image](https://user-images.githubusercontent.com/21112975/184249587-5862d01d-77ac-49a8-af12-3362cde1932a.png)

>Let us know if we can clarify anything on our end.
